src/main/resources/assets/opencomputers/lang/de_DE.lang

```diff
@@ -117,12 +123,14 @@ item.oc.UpgradeContainerCard2.name=Karten-Behälter (Stufe 3)
 item.oc.UpgradeContainerUpgrade0.name=Upgrade-Behälter (Stufe 1)
 item.oc.UpgradeContainerUpgrade1.name=Upgrade-Behälter (Stufe 2)
 item.oc.UpgradeContainerUpgrade2.name=Upgrade-Behälter (Stufe 3)
-item.oc.UpgradeCrafting.name=Fertigungs-Upgrade
+item.oc.UpgradeCrafting.name=Crafting-Upgrade
```

Comment:
Personally I found the wording a bit unclear. In all other cases you have some context and can guess that Fertigung refers to combining items in a crafting grid, but here there's nothing else to indicate what exactly this upgrade does (well, except for the icon). I found Crafting easier to understand here than Fertigung, but maybe that's just me.